linchpin/cli/__init__.py

```diff
@@ -401,6 +402,8 @@ def lp_up(self, targets=(), run_id=None, tx_id=None, inv_f="cfg"):
                     self._write_distilled_context(run_data)
         self._write_latest_run()
         self._write_to_inventory(inv_format=inv_f)
+        if ('post' in self.pb_hooks) and (self.__meta__ == "CLI"):
+            self.hook_state = '{0}{1}'.format('post', 'up')
```

Comment:
Are there down hooks as well?

Reply:
@p3ck yes. There's 4 specific hooks: 'preup', 'postup', 'predestroy', 'postdestroy'. You can see an example of them here: https://github.com/CentOS-PaaS-SIG/linchpin/blob/develop/docs/source/examples/workspace/PinFile.complex.yml#L22